lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/KBVStorageService.java

```diff
@@ -1,33 +1,28 @@
 package uk.gov.di.ipv.cri.kbv.api.service;
 
-import software.amazon.lambda.powertools.parameters.ParamManager;
 import uk.gov.di.ipv.cri.common.library.persistence.DataStore;
 import uk.gov.di.ipv.cri.common.library.persistence.DynamoDbEnhancedClientFactory;
 import uk.gov.di.ipv.cri.kbv.api.domain.KBVItem;
 
-import java.util.Objects;
-import java.util.Optional;
 import java.util.UUID;
 
+import static uk.gov.di.ipv.cri.kbv.api.config.ConfigurationConstants.KBV_TABLE_NAME;
+
 public class KBVStorageService {
     private final DataStore<KBVItem> dataStore;
 
     public KBVStorageService() {
-        this.dataStore =
+        this(
                 new DataStore<>(
-                        getKBVTableName(),
+                        new AWSParamStoreRetriever().getValue(KBV_TABLE_NAME),
```

Comment:
the ConfigurationService should be used to retrieve the database name (see the PersonIdentityService class for an example )
It might also be worthwhile adding a generic method to the ConfigurationService where the parameter name can be passed in as a string